apps/leo_manager/include/leo_manager.hrl

```diff
@@ -404,9 +429,77 @@
           id = 1 :: pos_integer(),
           node :: atom(),
           rebalance_info = [] :: list(tuple()),
-          timestamp = 0 :: pos_integer()
+          timestamp = 1 :: pos_integer()
+         }).
+
+
+-define(STATE_INVALID, 0).
+-define(STATE_PENDING, 1).
+-define(STATE_ONGOING, 2).
```

Comment:
Maybe adding a comment about what STATE_ONGOING (Finding objects to be deleted) means would be good for maintainers.